cltk/lemmatize/latin/unsupervised.py

```diff
@@ -0,0 +1,382 @@
+        onlyfiles = [join(path, f) for f in onlyfiles]
+        for filename in onlyfiles:
+            if '.tess' in filename:
+                print(filename)
+                self.read_files(filename)
+
+    def test_count_dictionary(self, token_list, lemma_list):
+        '''Test the ability of lemmatize(), (which uses the self.count_dictionary dictionary,
+        to predict the most likely lemmatization in ambiguous cases. Punctuation is
+        automatically counted as correct, because the 'punc' lemmatization usage is inconsistent
+        in the test corpus.
+        dependencies
+        ------------
+        itemgetter class from operator package
+        parameters
+        ----------
+        token_list: a list of tokens
+        lemma_list: a list of corresponding 'correct' lemmatizaitons
+        results
+        -------
+        prints four numbers: the number of correctly assigned lemmas in ambiguous cases;
+        the number of ambiguous cases in total; the number of tokens analyzed; and a
+        decimal between 0 and 1 representing the proportion of correct lemmatizations.
+        return
+        ------
+        a list object containing all incorrect lemmatizations for analysis. Format:
+        [(token, answer_given, correct_answer), (token...)]
+
+        NOTE: Initial tests show roughly 91% accuracy, identification of punctuation included.
+        '''
+        trials = 0
+        correct = 0
+        errors = []
+        for position in range(0, (len(token_list)-1)):
+            lemmalist = self.lemmatizer.lookup(token_list[position])
+            lemmalist = lemmalist[1]
+            lemma = max(lemmalist, key=itemgetter(1))
+            if len(lemmalist) > 1:
+                trials = trials + 1
+                if lemma[0] == lemma_list[position] or lemma[0] == 'punc':
+                    correct = correct + 1
+                else:
+                    errors.append((token_list[position], lemma[0], lemma_list[position]))
+        print(correct)
+        print(trials)
+        print(len(lemma_list))
+        rate = (len(lemma_list) - trials + correct) / len(lemma_list)
+        print(rate)
+        return errors
+
+
+class TessFile(object):
+    """Buffered/non-buffered reader for .tess file I/O.
+
+    Parameters
+    ----------
+    path : str
+        Path to the .tess file.
+    mode : str
+        File open mode ('r', 'w', 'a', etc.)
+    buffer : bool
+        If True, load file contents into memory on-the-fly. Otherwise, load in
+        contents on initialization.
+
+    Attributes
+    ----------
+    path : str
+        Path to the .tess file.
+    mode : str
+        File open mode ('r', 'w', 'a', etc.)
+    buffer : bool
+        If True, load file contents into memory on-the-fly. Otherwise, load in
+        contents on initialization.
+    hash : str
+        MD5 hash of the file.
+
+
+    """
```

Comment:
Prefer one space after the triple quote. The line breaks before the end can then be removed.